mathics/builtin/numbers/constants.py

```diff
@@ -207,7 +203,7 @@ class Catalan(_MPMathConstant, _SympyConstant):
     </dl>
 
     >> Catalan // N
-     = 0.915965594177219
+     = 0.915966
```

Comment:
Does this change in precision match Mathematica more closely? 

I am curious what causes the change in precision here?

Reply:
Yes: In WMA,
```
In[1]:= Catalan //N                                                             

Out[1]= 0.915966
```

Reply:
Great - thanks! Which "method" is getting used here. And what was it before?

Reply:
"sympy" is the default, but this is decided inside the `get_constant` method.